packages/braid-design-system/src/lib/components/Tag/Tag.screenshots.tsx

```diff
@@ -4,7 +4,7 @@ import { Tag, Inline, IconTag, Stack } from '../';
 import { debugTouchableAttrForDataProp } from '../private/touchable/debugTouchable';
 
 export const screenshots: ComponentScreenshot = {
-  screenshotWidths: [320],
+  screenshotWidths: [768],
```

Comment:
Increasing the screenshot width to avoid wrapping. `Tag` has no responsive behaviours and its more important to see alignment across variants horizontally.